packages/smooth_app/lib/pages/product/common/product_list_popup_items.dart

```diff
@@ -216,6 +222,112 @@ class ProductListPopupOpenInWeb extends ProductListPopupItem {
   }
 }
 
+/// Popup menu item for the product list page: export list.
+class ProductListPopupExport extends ProductListPopupItem {
+  @override
+  String getTitle(final AppLocalizations appLocalizations) =>
+      appLocalizations.product_list_export;
+
+  @override
+  IconData getIconData() => Icons.download;
+
+  @override
+  ProductListPopupMenuEntry getEntry() => ProductListPopupMenuEntry.export;
+
+  @override
+  bool isDestructive() => false;
+
+  @override
+  Future<ProductList?> doSomething({
+    required final ProductList productList,
+    required final LocalDatabase localDatabase,
+    required final BuildContext context,
+  }) async {
+    final String listName = ProductQueryPageHelper.getProductListLabel(
+      productList,
+      AppLocalizations.of(context),
+    );
+    final String fileName =
+        "${listName.replaceAll(' ', '-').toLowerCase()}_${DateTime.now().millisecondsSinceEpoch}.csv";
```

Comment:
iso8601 is more readable than time since epoch.

Reply:
iso8601 contains colons which are not allowed in filenames. Is it okay to break the standard and replace these characters?

Reply:
I know that there would be character replacements, but that would be much easier to read for the user.
Or, just use your own format in order to get something like `2025_01_29_10_22_59`.